components/bl-audio-player-component/README.md

```diff
@@ -1,25 +1,28 @@
 # Audio Player
 
-This is a component for Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer. You will have a styled player with the set of your tracks. It supports one or multiple audio sources and has all functions you can think of.
+The Audio Player component provides basic functionality to play audio files and streams.
 
-While the player can be configured with multiple settings it is easy in use and provides users with smooth experience.
+For more information about the configuration of this component, refer to the **Configuration** section below.
```

Comment:
Add please anchor link for this section

Here you could find how to do it
https://github.com/Backendless/ui-builder-library/pull/286

Reply:
Added